lib/sisimai/bite/email/kddi.rb

```diff
@@ -0,0 +1,166 @@
+module Sisimai::Bite::Email
+  module KDDI
+    # Sisimai::Bite::Email::KDDI parses a bounce email which created by au by KDDI.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/KDDI.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'from'       => %r/no-reply[@].+[.]dion[.]ne[.]jp/,
+        :'reply-to'   => %r/\Afrom[ \t]+\w+[.]auone[-]net[.]jp[ \t]/,
+        :'received'   => %r/\Afrom[ ](?:.+[.])?ezweb[.]ne[.]jp[ ]/,
+        :'message-id' => %r/[@].+[.]ezweb[.]ne[.]jp[>]\z/,
+      }
+      Re1 = {
+        :begin => %r/\AYour[ ]mail[ ](?:
+             sent[ ]on:?[ ][A-Z][a-z]{2}[,]
+            |attempted[ ]to[ ]be[ ]delivered[ ]on:?[ ][A-Z][a-z]{2}[,]
+            )
+        /x,
+        :rfc822 => %r|\AContent-Type: message/rfc822\z|,
+        :error  => %r/Could not be delivered to:? /,
+        :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReFailure = {
+        mailboxfull: %r/As[ ]their[ ]mailbox[ ]is[ ]full/x,
+        norelaying:  %r/Due[ ]to[ ]the[ ]following[ ]SMTP[ ]relay[ ]error/x,
+        hostunknown: %r/As[ ]the[ ]remote[ ]domain[ ]doesnt[ ]exist/x,
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'au by KDDI: http://www.au.kddi.com'; end
```

Comment:
__[rubocop]__
* Style/SingleLineMethods: Avoid single-line method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->